torch/optim/lr_scheduler.py

```diff
@@ -427,25 +427,78 @@ def _get_closed_form_lr(self):
                 for base_lr in self.base_lrs]
 
 
-class WarmUpLR(_LRScheduler):
-    """Decays the learning rate of each parameter group by either a small constant
-    or linearly increasing small warmup factor until the number of epoch reaches a
-    pre-defined milestone: warmup_iters. Notice that such decay can happen
-    simultaneously with other changes to the learning rate from outside this scheduler.
+class ConstantLR(_LRScheduler):
```

Comment:
We need to review the descriptions of this class because in some parts it still talks about warming up and "if the method is constant".

Reply:
thanks for pointing it out, i noticed it before and fixed in local version. but it seems that i haven't updated the remote version before.